docs/sandbox-permissions.rst

```diff
@@ -77,7 +77,8 @@ commonly require, and can therefore be freely used:
 - ``--share=network`` - access the network
 - ``--socket=x11`` - show windows using X11
 - ``--socket=fallback-x11`` - show windows using X11, if Wayland is not
-  available, overrides ``x11`` socket permission
+  available, overrides ``x11`` socket permission. Note that this does 
+  not obviate ``--socket=wayland``
```

Comment:
Sorry to nit-pick but I think `obviate` is not common vocabulary and it could be simplified for non-native english speakers. Such as "Note that you still must use ``--socket=wayland``"

Reply:
Ok, 60f6c6122be389592417b681e2cfa13dfb481df2 addresses this.